src/EFCore.PG/Infrastructure/NpgsqlDbContextOptionsBuilder.cs

```diff
@@ -68,6 +68,24 @@ public virtual void UseAdminDatabase([CanBeNull] string dbName)
         public virtual void SetPostgresVersion([CanBeNull] Version postgresVersion)
             => WithOption(e => e.WithPostgresVersion(postgresVersion));
 
+        /// <summary>
+        /// Maps a user-defined PostgreSQL range type for use with all DbContexts.
+        /// </summary>
+        /// <param name="rangeName">The name of the PostgreSQL range type to be mapped.</param>
+        /// <param name="elementClrType">
+        /// The CLR type of the range's element. The actual mapped type will be an <code>NpgsqlRange{T}</code> over
+        /// this type.
+        /// </param>
+        /// <param name="subtypeName">
+        /// Optionally, the name of the range's subtype or element. This is usually not needed - the subtype will be
+        /// inferred based on <paramref name="elementClrType"/></param>
+        /// <example>
+        /// To map a range of PostgreSQL real, use the following:
+        /// <code>NpgsqlTypeMappingSource.MapRange("real", typeof(float));</code>
+        /// </example>
+        public virtual void MapRange([NotNull] string rangeName, [NotNull] Type elementClrType, string subtypeName = null)
+            => WithOption(e => e.WithRangeMapping(rangeName, elementClrType, subtypeName));
```

Comment:
I think it makes sense to add a generic version of the method.

Reply:
Done. Also renamed from element to subtype, which is the PostgreSQL naming for ranges (unlike arrays).

Also corrected comments and variable names from `element` to `subtype`.